pallets/messages/src/lib.rs

```diff
@@ -376,7 +376,11 @@ impl<T: Config> Pallet<T> {
 		// insert into storage and create events
 		for (schema_id, messages) in map {
 			let count = messages.len() as u16;
-			let bounded_vec: BoundedVec<_, _> = messages.try_into().unwrap();
+			let bounded_vec: BoundedVec<_, _> = messages.try_into().unwrap_or_default();
```

Comment:
This is probably where we want to add a log.

Reply:
Maybe we add something along this line so that we can log and also return the proper weight in the case of an error. Something along this lines:
```
match message.try_into() {
  Ok() => {
    if bounded_vec.is_empty() { return ...}
  },
  Error() => {
log("something")
   return T::DbWeight::get().reads(1)
     }
   
}
```

Reply:
Actually, probably don't want this either.

Reply:
Me likey, question is do we have any holes that this case may occur, "terrible"

Reply:
lol...iI think it autocorrected to terrible and I just rolled with it. Terrible is probably not the right word.

Reply:
Yeah, I was kind of on the fence about this anyway, a log is a better idea.